pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.igniterealtime.openfire.plugins</groupId>
     <artifactId>jsxc</artifactId>
-    <version>4.2.0.2-SNAPSHOT</version>
+    <version>4.2.1.2-SNAPSHOT</version>
```

Comment:
The version identifier uses four digits. The convention that I use here (and also for the inVerse plugin) 
- the first three reflect the version of JSXC that the plugin is based on.
- the last digit is a counter of releases of the Openfire plugin based on that version of JSXC.

As this commit updates the version of JSXC from 4.2.0 to 4.2.1, and we obviously do not have a previous release of this plugin that's based on that JSXC version, the version number should be a concatination of `4.2.0` and `1`: `4.2.1.1`

Reply:
Ignore the above. You did this in distinct commits from which just looking at the result. All is well.